CMakeLists.txt

```diff
@@ -324,7 +324,8 @@ write_basic_package_version_file(
 
 install(TARGETS yaml-cpp 
     EXPORT "${targets_export_name}"
-    INCLUDES DESTINATION "${include_install_dir}"
```

Comment:
There is no need to touch this part.

Reply:
I am trying to undo the formatting but my editor is making it hard!